how to deploy datasources?

I have created a file:

```
{
  grafanaDatasources+:: {
    'rel-perfil-api.yml': {
      access: 'proxy',
      isDefault: true,
      jsonData: {
        httpMethod: 'GET',
      },
      name: 'rel perfil',
      type: 'prometheus',
      url: 'http://prd-ath-barimetrics-monitor-plat-re.ath-barimetrics-monitor-plat.svc.cluster.local',
    },
  },
}
```

Use the command `grr apply datasource.libsonnet` and get the error:

```
panic: interface conversion: interface {} is map[string]interface {}, not string

goroutine 1 [running]:
github.com/grafana/grizzly/pkg/grafana.(*DashboardHandler).Parse(0x1b47588, 0xc000023140, 0x16, 0x14e1b60, 0xc00020ec00, 0x0, 0x0, 0x0)
	/Users/gustavocoelho/Documents/dashboards/grizzly/pkg/grafana/dashboard-handler.go:84 +0x5f4
github.com/grafana/grizzly/pkg/grizzly.Parse(0xc00006b6e0, 0x1, 0x1, 0xc000120780, 0x3, 0x4, 0xc0001d22a0, 0xc0001d22d0, 0x0, 0x0, ...)
	/Users/gustavocoelho/Documents/dashboards/grizzly/pkg/grizzly/workflow.go:118 +0x761
main.applyCmd.func1(0xc000139e00, 0xc00006b8f0, 0x1, 0x1, 0x0, 0x0)
	/Users/gustavocoelho/Documents/dashboards/grizzly/cmd/grr/workflow.go:89 +0xc4
github.com/go-clix/cli.(*Command).execute(0xc000139e00, 0xc00006b8c0, 0x1, 0x1, 0xc000139e00, 0xc00006b8c0)
	/Users/gustavocoelho/go/pkg/mod/github.com/go-clix/cli@v0.1.0/command.go:118 +0x284
github.com/go-clix/cli.(*Command).Execute(0xc000139900, 0xc0001c3ee0, 0x9)
	/Users/gustavocoelho/go/pkg/mod/github.com/go-clix/cli@v0.1.0/command.go:74 +0x145
main.main()
	/Users/gustavocoelho/Documents/dashboards/grizzly/cmd/grr/main.go:47 +0x41c
```

How should I deploy a datasource?

--- a/pkg/grafana/dashboard-handler.go
+++ b/pkg/grafana/dashboard-handler.go
@@ -81,10 +81,12 @@ func (h *DashboardHandler) newDashboardFolderResource(path, folderName string) g
 func (h *DashboardHandler) Parse(path string, i interface{}) (grizzly.ResourceList, error) {
 	resources := grizzly.ResourceList{}
 	if path == dashboardFolderPath {
-		folderName := strings.ReplaceAll(i.(string), "{ }", "") // No idea why json parsing adds { } to the end of the parsed string :-(
-		resource := h.newDashboardFolderResource(path, folderName)
-		resources[dashboardFolderPath] = resource
-		return resources, nil
+		if _, ok := i.(string); ok {
+			folderName := strings.ReplaceAll(i.(string), "{ }", "") // No idea why json parsing adds { } to the end of the parsed string :-(
+			resource := h.newDashboardFolderResource(path, folderName)
+			resources[dashboardFolderPath] = resource
+			return resources, nil
+		}
 	}
 	msi := i.(map[string]interface{})
 	for k, v := range msi {

